front-end/src/Header.js

```diff
@@ -0,0 +1,25 @@
+import 'react-sticky-header/styles.css';
+import StickyHeader from 'react-sticky-header';
+import './Header.css' 
+import Hamburger from './Hamburger'
+
+const Header = () => (
+  [
+  <Hamburger pageWrapId={'page-wrap'} outerContainerId={'outer-container'} />,
+  <StickyHeader    
+    // This section is sticky 
+    header={
+      <div className="Header_root">
+        <h1 className="Header_title">TV Tracker</h1>
+      </div>
+    }
+  >
+    <section>
+      <p><br></br><br></br><br></br><br></br></p>
+    </section>
```

Comment:
I think with some css/styling work done (as mentioned above), this could be removed! My impression, from what I've seen online, is that it's better practice to space out elements using css rather than inserting `br` elements.

Reply:
If possible, can you help me with this on Sunday too, please?